api/campaigns/index.js

```diff
@@ -89,6 +90,19 @@ router.get('/camp/:id', (req, res) => {
     });
 });
 
+router.get('/:id/submissions', async (req, res) => {
+  
+  const { id } = req.params;
+
+  await ApplicationSubmissions.findAllByCampaignId(id)
```

Comment:
nit: you're already using async/await here, get rid of .then and .catch, replacing them with a variable (`const applicationSubmissions = await ApplicationSubmissions.findAllByCampaignId(id)`) and surrounding the whole thing in try/catch